package.json

```diff
@@ -18,7 +18,7 @@
         "lint": "eslint . --ext .ts,.tsx",
         "lint:fix": "eslint . --ext .ts,.tsx --fix",
         "install:all": "npm i && cd packages/webapp && npm i && cd ../..",
-        "ts-build": "tsc -b --clean packages/shared packages/server packages/cli packages/runner packages/jobs && tsc -b tsconfig.build.json && tsc -b packages/webapp/tsconfig.json",
+        "ts-build": "tsc -b --clean packages/shared packages/server packages/cli packages/runner packages/jobs && tsc -b tsconfig.build.json && npm run postbuild -ws --if-present && tsc -b packages/webapp/tsconfig.json",
```

Comment:
What is this doing?

Reply:
run the `postbuild` script for all packages if present. so doing `npm run ts-build` to build all packages or `npm run build -w nango` behaves the same. For example, copy the `sync.d.ts` type file for the CLI regardless of how the package is being built